### ISSUE TITLE
Make .md5 checksum files compatible with linux

### ISSUE THREAD

**Issue by heldchen** [gurnec/HashCheck#36]
HashCheck saves the md5 checksum file with CRLF line endings, which the linux md5sum tool cannot properly handle. Please add an option to save the checksum files with LF line endings.

**Comment by R-Adrian** [comment]
add a call to dos2unix in your script. it will only change the file if it has CRLF endings, otherwise it does not change the file.

**Comment by heldchen** [comment]
yeah I know the workarounds. nevertheless hashcheck should have a way to save the file properly.

**Comment by spgreene** [comment]
I've been using HashCheck for a while and have gotten in the habit of manually changing the generated file to use LF instead of CRLF. Also, the `*sum` tools in Linux use two-spaces (`  `) instead of the space asterisk (` *`) to separate the hash and filename. 

For reference, using the space-space separator with LF (Unix line endings) in my checksum file seems to have no effect on HashCheck in Windows, so there doesn't seem to be much of a downside in changing this output format that I can see (given my very limited view).
